addons/field-guide/testem.js

```diff
@@ -1,23 +0,0 @@
-'use strict';
```

Comment:
Reading the PR my assumption is this relates to "reduce the code to the bare minimum" and would explain why some other default ember-cli files are also deleted in this PR. I would personally recommend keep these files around so that we maintain consistency with a conventional ember-cli addon.

Reply:
the idea is exactly this: reduce everything to a bare minimum (I also tried to remove entirely the addon and have everything inside the website, but it I have to check with Matthew Irish a couple of things to see if it's possible)

If it's not possible, then (after the launch) we can discuss what to make of this part of the code (the one under the `addon/field-guide`).